### ISSUE TITLE
Object and nested fields broken in 5.2.11

### ISSUE THREAD

**Issue by maxcom** [sksamuel/elastic4s#819]
Object/nested field mappings syntax was changed in 5.2.11. I updated mapping definition from 

```
field(destination, ObjectType) as (
        keywordField(destination.address),
        keywordField(destination.person)
      )
```

to

```
objectField(destination) fields (
        keywordField(destination.address),
        keywordField(destination.person)
      ),
```

and no it does not work:

```
org.elasticsearch.index.mapper.MapperParsingException: Exception raised on property evaluation.
> ARG_0: de4406f6-90c6-4130-b26c-e5d83f790806
> Exception: org.elasticsearch.index.mapper.MapperParsingException: Failed to parse mapping [event]: Mapping definition for [destination] has unsupported parameters:  [fields : {address={type=keyword}, person={type=keyword}}]
> caused by MapperParsingException[Mapping definition for [destination] has unsupported parameters:  [fields : {address={type=keyword}, person={type=keyword}}]]
MapperParsingException[Failed to parse mapping [event]: Mapping definition for [destination] has unsupported parameters:  [fields : {address={type=keyword}, person={type=keyword}}]]; nested: MapperParsingException[Mapping definition for [destination] has unsupported parameters:  [fields : {address={type=keyword}, person={type=keyword}}]];

```

**Comment by sksamuel** [comment]
Ok I see, I need to change object field to be like nested field and use `properties` not `fields` for subfields.

**Comment by sksamuel** [comment]
Also need to restore `as` as an alias for fields on these types.

**Comment by sksamuel** [comment]
Fixed will be in 5.2.12

**Comment by rodrigopr** [comment]
Hi @sksamuel, when is the version 5.2.12 schedule to release?
